nmostesting/suites/IS0401Test.py

```diff
@@ -1088,10 +1088,12 @@ def test_19(self, test):
                 if interface_name in interfaces:
                     return test.FAIL("Duplicate interface name '{}' found in Node API self resource"
                                      .format(interface_name))
-                interfaces.add(interface_name)
+                interfaces[interface_name] = interface

```

Comment:
Could the warning check not be done here, rather than handling it whilst checking things more relevant to senders and receivers? That would avoid the need to store the extra interface object.

Reply:
I wanted to avoid checking interfaces that might only be used for management functions.

Reply:
I see. Is there not potentially a danger that an interface may not be attached to a Sender/Receiver at the time of testing?

Reply:
We can only test what is in front of us though? If the Node-under-test isn't fully operational, e.g. one interface isn't connected, or it doesn't yet have any Senders or Receivers configured, we can't test those.